app/commands/github/task/create_or_update.rb

```diff
@@ -0,0 +1,37 @@
+module Github
+  class Task
+    class CreateOrUpdate
+      include Mandate
+
+      initialize_with :issue_url, :attributes
+
+      def call
+        task = ::Github::Task.create_or_find_by!(issue_url: issue_url) do |t|
+          t.title = attributes[:title]
+          t.repo = attributes[:repo]
+          t.opened_at = attributes[:opened_at]
+          t.opened_by_username = attributes[:opened_by_username]
+          t.action = attributes[:action]
+          t.knowledge = attributes[:knowledge]
+          t.area = attributes[:area]
+          t.size = attributes[:size]
+          t.type = attributes[:type]
+        end
```

Comment:
I think you can just do `t.attributes = attributes` here, as the LHS and RHS are identical.